src/Control/Biegunka/Source/Git.hs

```diff
@@ -97,29 +93,43 @@ git_ :: URI -> FilePath -> Script 'Sources ()
 git_ u p = git' u p mempty
 
 updateGit :: URI -> FilePath -> Git -> IO (Maybe String)
-updateGit u p Git { _remote, _branch } =
+updateGit u p Git { _branch, _failIfAhead } =
   doesDirectoryExist p >>= \case
     True -> do
-      let rbr = _remote </> _branch
+      let rbr = "origin" </> _branch
       before <- gitHash
+      remotes <- askGit p ["remote"]
```

Comment:
Can you move the action's output post-processing closer to the action itself? E.g.
```
remotes <- fmap lines (askGit p ["remote"])
when ("origin" `notElem` remotes) $
  ...
```
It's quite a bit easier to understand the code that way (for me, at least).